graylog2-server/src/main/java/org/graylog2/indexer/messages/Messages.java

```diff
@@ -67,49 +90,44 @@ public Messages(Client client, ElasticsearchConfiguration configuration) {
     }
 
     public ResultMessage get(String messageId, String index) throws IndexMissingException, DocumentNotFoundException {
-		GetRequestBuilder grb = new GetRequestBuilder(c, index);
-		grb.setId(messageId);
-
-		GetResponse r = c.get(grb.request()).actionGet();
-		
-		if (!r.isExists()) {
-			throw new DocumentNotFoundException();
-		}
-		
-		return ResultMessage.parseFromSource(r);
-	}
-	
-	public List<String> analyze(String string, String index) throws IndexMissingException {
-		List<String> tokens = Lists.newArrayList();
-		AnalyzeRequestBuilder arb = new AnalyzeRequestBuilder(c.admin().indices(), index, string);
-		AnalyzeResponse r = c.admin().indices().analyze(arb.request()).actionGet();
-		
-		for (AnalyzeToken token : r.getTokens()) {
-			tokens.add(token.getTerm());
-		}
-		
-		return tokens;
-	}
+        GetRequestBuilder grb = new GetRequestBuilder(c, index);
+        grb.setId(messageId);
+
+        GetResponse r = c.get(grb.request()).actionGet();
+
+        if (!r.isExists()) {
+            throw new DocumentNotFoundException();
+        }
+
+        return ResultMessage.parseFromSource(r);
+    }
+
+    public List<String> analyze(String string, String index) throws IndexMissingException {
+        List<String> tokens = Lists.newArrayList();
+        AnalyzeRequestBuilder arb = new AnalyzeRequestBuilder(c.admin().indices(), index, string);
+        AnalyzeResponse r = c.admin().indices().analyze(arb.request()).actionGet();
+
+        for (AnalyzeToken token : r.getTokens()) {
+            tokens.add(token.getTerm());
+        }
+
+        return tokens;
+    }
 
     public boolean bulkIndex(final List<Message> messages) {
         if (messages.isEmpty()) {
             return true;
         }
 
-        final BulkRequestBuilder request = c.prepareBulk();
+        final BulkRequestBuilder requestBuilder = c.prepareBulk().setConsistencyLevel(WriteConsistencyLevel.ONE);
```

Comment:
Is this consistency level change intentional? How does this change the behaviour?